### PR TITLE
LOG-690: vendor curator5

### PR DESCRIPTION
**Pull request by jcantrill**
This PR
* bumps the curator version to work with es6.x
* vendors curator and its dependencies to facilitate downstream builds
* adds rh-manifest.txt

cc @sfowl

### CODE REVIEW

File: curator/Dockerfile [hunk 1]
```diff
@@ -13,42 +13,37 @@ ENV HOME=/opt/app-root/src \
...
     chgrp -R 0 ${HOME} && \
-    chmod -R g=u ${HOME}
+    chmod -R g=u ${HOME} && \
+    cd $HOME/vendor && \
+    for pkg in $(ls *.tar.gz); do \
+      pip install --no-deps $pkg; \
+    done  && \
+    tar -xf python-dateutil-2.8.1.tar.gz && \
+    mv python-dateutil-2.8.1/python_dateutil.egg-info python_dateutil-2.8.2-py2.7.egg-info && \
+    mv python_dateutil-2.8.2-py2.7.egg-info /usr/lib/python2.7/site-packages && \
```
Comment (periklis) [inline curator/Dockerfile:43]: Maybe renaming 2.8.1 to 2.8.2 a typo?

File: curator/Dockerfile [hunk 2]
```diff
@@ -13,42 +13,37 @@ ENV HOME=/opt/app-root/src \
...
+    CURATOR_VER=5.8.1 \
     container=oci
 
-ARG LOCAL_REPO
-RUN if [ -n "${LOCAL_REPO}" ] ; then \
-     curl -s -o /etc/yum.repos.d/local.repo ${LOCAL_REPO} ; \
-    fi
+LABEL io.k8s.description="Curator elasticsearch container for elasticsearch deletion/archival" \
+  io.k8s.display-name="Curator ${CURATOR_VER}" \
+  io.openshift.tags="logging,elk,elasticsearch,curator"
```
Comment (periklis) [inline curator/Dockerfile:22]: Does shrinking the labels here comply with https://github.com/projectatomic/ContainerApplicationGenericLabels#overview?

File: curator/rh-manifest.txt [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+idna==2.9
+jmespath==0.9.5
+python-dateutil==2.8.1
+PyYAML==3.13
+requests==2.23.0
+requests-aws4auth==0.9
+s3transfer==0.3.3
+six==1.14.0
+urllib3==1.24.2
+voluptuous==0.11.5
```
Comment (periklis) [inline curator/rh-manifest.txt:19]: Does the file format comply with https://mojo.redhat.com/docs/DOC-1195158?

File: curator/run.sh [hunk 4]
```diff
@@ -22,11 +22,11 @@ function waitForES() {
   exit 255
 }
 
-waitForES
+#waitForES
```
Comment (periklis) [inline curator/run.sh:25]: Is this disabled by accident?

File: curator/rh-manifest.txt (hunk 3 above)
Comment (jcantrill) [reply curator/rh-manifest.txt:19]: Per the doc there really is no specific file format, but received buy in from the person who asked me to add it

File: curator/Dockerfile (hunk 2 above)
Comment (jcantrill) [reply curator/Dockerfile:22]: These are the only ones that will get built into the image using the current art process: `Import description, tags, display-name from the labels in their Dockerfile` https://mojo.redhat.com/docs/DOC-1182562

File: curator/Dockerfile.centos7 [hunk 5]
```diff
@@ -13,28 +13,36 @@ ENV HOME=/opt/app-root/src \
...
+COPY . ${HOME}
 
 RUN mkdir -p $(dirname "$CURATOR_CONF_LOCATION") && \
     touch ${CURATOR_CONF_LOCATION} && \
     chmod -R u+x ${HOME} && \
     chgrp -R 0 ${HOME} && \
-    chmod -R g=u ${HOME}
+    chmod -R g=u ${HOME} && \
+    cd $HOME/vendor && \
+    for pkg in $(ls *.tar.gz); do \
```
Comment (ewolinetz) [inline curator/Dockerfile.centos7:38]: is using `.tar.gz` files something ART is going to be okay with?
cc @sosiouxme

**Comment by ewolinetz** [comment]
/hold
this was auto approved

File: curator/Dockerfile.centos7 (hunk 5 above)
Comment (ewolinetz) [reply curator/Dockerfile.centos7:38]: confirmed on slack that this is okay

**Pull request merged**